app/components/ui/search/index.js

```diff
@@ -158,6 +165,10 @@ const Search = React.createClass( {
 	},
 
 	render() {
+		if ( true ) {
```

Comment:
I think we can remove this completely. I mean, the `if ( true )` as we don't care about the linter at this point.